c7n/resources/code.py

```diff
@@ -142,6 +142,16 @@ class resource_type(TypeInfo):
     }
 
 
+@resources.register('codebuild-source-credentials')
```

Comment:
thoughts on just saying credential or token (per arn identifier)

Reply:
will "codebuild-credential" be appropriate?

Reply:
Anyway we should indicate that this resource is related to codebuild service

Reply:
credential works